package.json

```
@@ -6,9 +6,11 @@
   "license": "MIT",
   "scripts": {
     "generate-data": "node generateData.js",
-    "add-data": "git add universe.json"
+    "add-data": "git add universe.json",
+    "new:lib": "node newLib.js"
```

minor nitpicking
```suggestion
    "lib:new": "node newLib.js"
```
can you update also the way it is on the hygen file structure? `lib/new.ejs`, it's the pattern we are using already in other projects
will make it easier to add more namespaces + verbs later

---

one .ejs.t file to create all others files?